### PR TITLE
Project: News-Web

### PR DESCRIPTION
**Pull request by sammyifelse**
# Hackathon Submission: <PROJECT_NAME>

## Name of Team Lead
_Enter your full name_

## GitHub handles of Team Members  
_Enter the GitHub handles of all your team members (including yourself) as separate bullet points_

### Are all of your team members registered on our [Hackathon Signup page](https://apwr.dev/htf24-hackathon)?
_This is a mandatory step for all participants to be considered eligible for judging_

- [x] Yes
- [ ] No

---

## Rules and Code of Conduct  
By submitting this PR, you agree to follow our Rules and Code of Conduct.

- [x] Yes

---

## Anything Else?  
_Any other feedback, queries or information you would like to share with us?_

### CODE REVIEW

**Comment by adityaoberai** [comment]
@sammyifelse thanks a lot for submitting your PR! You can continue making updates to your PR and we'll review and merge these closer to the end of the month

**Review by adityaoberai — CHANGES_REQUESTED** [review]
@sammyifelse please make a copy of the `template_submission` file instead of editing it

**Comment by sammyifelse** [comment]
so i have to make a new file named template_submission?

**Comment by adityaoberai** [comment]
> ok i have made a new template_submission1.md file where i have copied the content of my project istead of editing the main template_submission.md file

You can rename the file to your project's name instead of `template_submission1.md`

**Comment by sammyifelse** [comment]
Done

**Comment by CodeWithKamal** [comment]
sorry guys, accidental clicked caused closed PR. how to undo this? I tried deleting that branch. @adityaoberai

**Comment by sammyifelse** [comment]
PR is not merged till now??

File: .vscode/settings.json
Comment (adityaoberai) [inline .vscode/settings.json:1]: Please remove this file from the PR @sammyifelse

**Review by adityaoberai — CHANGES_REQUESTED** [review]

**Review by adityaoberai — APPROVED** [review]

**Comment by adityaoberai** [comment]
@sammyifelse thank you for your submission. Unfortunately, it is ineligible for this hackathon because, as visible in the project's  [commit history](https://github.com/sammyifelse/News-Web/commits/master/), the project has been made before the hackathon began.

image

As stated in the rules, the project must be built in its entirety within the month of October only. Older projects are not allowed. You can build and submit another project if you like, however, we will not be accepting this one.

**Pull request closed without merging**